crates/net/network/tests/it/connect.rs

```diff
@@ -302,8 +313,13 @@ async fn test_connect_to_trusted_peer() {
 
     dbg!(&headers);
 
-    while let Some(ev) = events.next().await {
-        dbg!(ev);
+    while let Some(result_ev) = events.next().await {
+        match result_ev {
+            Ok(ev) => {
+                dbg!(ev);
+            }
+            Err(err) => eprintln!("{err}"),
```

Comment:
let's use `tracing` crate for all logging! in tests we use `reth_tracing`, and start it with function `reth_tracing::init_test_tracing`

Reply:
sure, there were a few other occurrences of `eprintln!` in that file, replaced all of them with `error!` from `reth_tracing::tracing`